.github/workflows/macos-microshift.yml

```diff
@@ -35,6 +35,18 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: ${{ matrix.go }}
+      - name: check if OSP_PULL_SECRET exists
+        shell: bash
+        run: |
+          if [ "${{ secrets.OSP_PULL_SECRET }}" == '' ]; then
```

Comment:
You said in the other PR that this is not valid, that this has to be an env var?

Reply:
@cfergeau I was looking at the docs https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-secrets but when I tried out this does let me access the secrets context. I updated the `echo` also let's hope that also works.

Reply:
And workflow job is failing so I think it is better to use documented flow instead using secret context directly from if or from echo command.